build.gradle.kts

```diff
@@ -46,6 +46,7 @@ object Versions {
 }
 
 dependencies {
+    implementation(files("./local_lib/proto-kotlin-feature_msg-based-fees-proto-25a2450.jar"))
```

Comment:
hopefully this is published somewhere now?

Reply:
it is .. Ergels is publishing to maven central..i gotta check it out.